linker-interface/shared/src/test/scala/org/scalajs/linker/interface/FingerprintBuilderTest.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.linker.interface
+
+import org.junit.Test
+import org.junit.Assert._
+
+import Fingerprint._
+
+class FingerprintBuilderTest {
+  @Test
+  def noFields(): Unit = {
+    val fingerprint = new FingerprintBuilder("Test0").build()
+    assertEquals("Test0()", fingerprint)
+  }
+
+  @Test
+  def oneField(): Unit = {
+    val fingerprint = new FingerprintBuilder("Test1")
+      .addField("test1", true)
+      .build()
+    assertEquals("Test1(test1=true)", fingerprint)
+  }
+
+  @Test
+  def multipleFields(): Unit = {
+    val test1: Option[Int] = Some(1)
+    val test2: Option[Int] = None
```

Comment:
I think it is easier to read if you type ascribe these at call site (if it is necessary to make implicit resolution work). E.g. `addField("test1", Some(1): Option[Int])`.

Otherwise this leaves me wondering why these are vals.

Reply:
Yes, those are indeed needed for implicit resolution. Fixed in eb6e308689224c3bbcc9c059e10fa1f02bee3acb